diffkemp/simpll/passes/ControlFlowSlicer.cpp

```diff
@@ -15,6 +15,7 @@
 
 #include "ControlFlowSlicer.h"
 #include "Utils.h"
+#include <Config.h>
```

Comment:
Include DiffKemp headers using `#include "..."`. Also, you'll have to move it one line up.